### DISCUSSION TITLE
Making Admin UI only accessible from specific subnet

### DISCUSSION THREAD

**Discussion by markhaehnel** [dani-garcia/vaultwarden#1411] [General]
Hey everyone,
i'm thinking of making the Admin UI only accessible from my local subnet. At the moment it is available from everywhere.

- What do you think about that?
- Is it possible with tools that bitwarden_rs already brings?
- Should i block outside access with a reverse proxy? (Not the first solution i would try because of the limited configurability of my Synology NAS' reverse proxy)

**Comment by BlackDex** [comment]
I'm not a big fan of building this into bitwarden_rs it self. And rather use a reverse proxy for this like nginx, caddy or treafik for example.

With some (if not all) you can also add an extra basic auth login infront off the /admin url if you want.

**Reply by markhaehnel** [reply]
I now think of adding an reverse proxy layer to my docker-compose configuration.
With that i can implement the sub-directory "filtering" there and keep the Synology managed reverse proxy for TLS termination.

//EDIT:
I've implemented it with Caddy2 in-between the Synology reverse proxy and bitwarden_rs. Works good so far!

Thank you for your work!
